types/string-algorithms/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for string-algorithms 1.0
+// Project: https://github.com/burgaard/string-algorithms
+// Definitions by: lionel-rowe <https://github.com/DefinitelyTyped>
```

Comment:
this has to be a real account handle, sorry.

Reply:
Fixed, thanks!